.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "bower_components"]
```

Comment:
oof, do we have to use a submodule? I was so happy to get rid of our submodule in the notebook.